_posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "Escaping The Tar Pit at RubyConf 2019"
+date: 2019-12-03 10:00:00
+categories: ["quality"]
```

Comment:
@etagwerker Let's use `code-quality` to be consistent (re: https://www.fastruby.io/blog/tags/code-quality)

```suggestion
categories: ["code-quality"]
```